src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1515,6 +1529,28 @@ private void OnCompileDotnetScript(MessageId msg_id, CancellationToken token)
             SendResponse(msg_id, Result.OkFromObject(new { }), token);
         }
 
+        private async Task<bool> OnContinueToLocation(MessageId sessionId, string script_id, int lineNumber, CancellationToken token)
+        {
+            DebugStore store = await RuntimeReady(sessionId, token);
+            ExecutionContext context = GetContext(sessionId);
+            Frame scope = context.CallStack.First<Frame>();
+            if (!SourceId.TryParse(script_id, out SourceId id))
+                return false;
+
+            var res = new List<SourceLocation>();
+
+            var sourceLocation = new SourceLocation(id, lineNumber, -1);
+
+            store.AddPossibleBreakpointsInMethodToList(sourceLocation, sourceLocation, res, scope.Method.Info);
+            var ilOffset = res.First().IlLocation;
+
+            var ret = await context.SdbAgent.SetNextIP(scope.Method, context.ThreadId, ilOffset, token);
+            var breakpointId = await context.SdbAgent.SetBreakpoint(scope.Method.DebugId, ilOffset.Offset, token);
+            context.BreakpointToDisable = breakpointId;
```

Comment:
If `context.BreakpointToDisable` is already set, then should it be removed first?

Reply:
I think this cannot happen because we can't continueToLocation twice without waiting for the debugger to be paused in the place that you choose. I mean, it will always call OnContinueToLocation and then the code where we remove the breakpoint before calling OnContinueToLocation again.